src/conda_actions.ts

```diff
@@ -284,12 +284,23 @@ const install_python = async (config: ConfigObject): Promise<void> => {
         '__setup_conda',
         `python=${config.python_version}`,
       ])
+    } else if (python_version.match(/^pypy((3\.\d+)?(=(\d+)?(\.\d+)?(\.\d+)?)?)?$/) !== null) {
+      await exec.exec('conda', [
+        'create',
+        '-y',
+        '-n',
+        '__setup_conda',
+        `config.python_version`,
```

Comment:
```suggestion
        `${config.python_version}`,
```
The [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals?retiredLocale=de) also needs to be evaluated right now it is just a hardcoded 'config.python_version' and will break